starter-code/src/components/Facebook.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+
+import Idcard from './IdCard.js';
+
+import './Facebook.css'
+
+class Facebook extends Component {
+  state = {
+    countryFilter: ''
+  }
+
+  setCountryFilter = (e) => {
+    this.setState({
+      countryFilter: e.target.innerHTML
+    })
```

Comment:
Your question is how to filter this properly.  There are of course many approaches one can take.  I think this is a good start, because you don't want to change the array but only target elements that satisfy a condition, so I don't think you need to store an array of filtered items.  But then your id cards might need an additional property `selected` or something, which your could react to inside that component.   So in you data.map where you render the IdCards you could have an expression or a function that evaluated multiple possible condition that you might add later : 
` selected={ profile.country == this.state.filterCondition || this.setSelectedIfNeeded() }`